Chrome extension: Add options page

Rob--W opened the pull request:
Fixes #4560

I've always wanted an options page, but there was little progress due to the lack of UI guidelines.
Starting with Chrome 40, extension options pages will be embedded at Chrome's settings page, which encourages the use of Chrome's UI styles. To view the options page on the current stable version of Chrome (39) with the new styles, start Chrome with the --enable-embedded-extension-options flag.

Screenshot:

![pdfjs-crx-options](https://cloud.githubusercontent.com/assets/1365071/5632784/d3069e3a-95d1-11e4-8cf0-6c31e26e6a0a.png)

Rob--W committed:
Chrome extension: Add options page

timvandermeij commented:
This looks really nice! I have tested this in Chrome 39 and I have highlighted some of the issues I have found above. In one list:

- Can we add some spacing between the last option and the reset button?
- Disabling 'Show previous position upon load' does not seem to work. The viewer will still go back to the original position for me.
- Setting the zoom level dropdown to 400% makes the viewer display pages at 4%.
- 'Sidebar state on load' does not appear to work. The sidebar always remains closed for me.

Rob--W commented:
@timvandermeij 

- No, the spacing is already consistent with Chrome's styles. Here is a screenshot where the PDF.js options are displayed in the background, and Chrome's settings page semi-transparantly on top. Note that the checkboxes/buttons are spaced and aligned correctly:
![pdfjs-chrome-options-and-chrome-settings](https://cloud.githubusercontent.com/assets/1365071/5720280/53753d10-9b22-11e4-8287-cbaa81e522fe.png)
- I've debugged this issue, and it is not specific to the Chrome extension. It is caused by the fact that the history's state is preserved when the page is reloaded. The page size, etc. are persisted in `history.state` across reloads.
- Fixed by multiplying the option values by 100.
- Fixed by using parseInt.

timvandermeij commented:
Awesome work! Tested and verified that my points were addressed. I will open a follow-up issue for the history state issue as it is not specific to this patch.
